readthedocs/doc_builder/director.py

```diff
@@ -335,6 +339,163 @@ def run_build_job(self, job):
         for command in commands:
             environment.run(*command.split(), escape_command=False, cwd=cwd)
 
+    def run_build_commands(self):
+        cwd = self.data.project.checkout_path(self.data.version.slug)
+        environment = self.vcs_environment
+        for command in self.data.config.build.commands:
+            environment.run(*command.split(), escape_command=False, cwd=cwd)
+
+        # Copy files to artifacts path so they are uploaded to S3
+        target = self.data.project.artifact_path(
+            version=self.data.version.slug,
+            type_="sphinx",
+        )
+        artifacts_path = os.path.join(cwd, "output")
+        if not os.path.exists(artifacts_path):
+            raise BuildUserError(BuildUserError.BUILD_COMMANDS_WITHOUT_OUTPUT)
+
+        shutil.copytree(
+            artifacts_path,
+            target,
+        )
+
+        self.data.outcomes = defaultdict(lambda: False)
+        self.data.outcomes["html"] = True
+
+    def install_build_tools(self):
```

Comment:
Would be useful to note in a comment that this was moved & not changed, since it's hard to know if it was changed when reviewing. I was also surprised that it was here, but then realized it was a refactor.

Reply:
I am a strong +1 on separating moving files, big linting, and similar operations into independent commits. Review is much easier if moving code is isolated from changing code. I'd even be +1 on setting a policy on this, as it does take a good deal of energy from reviewers.

Reply:
Yeah, 😞 . Santos asked the same in his first comment. It was just moved without changes in the logic.